### PR TITLE
fix: [BUG] elf family name not generating

### PR DESCRIPTION
**Pull request by Jedikay**
Line 61 and 70, added code to generate random family name.

### CODE REVIEW

**Comment by W5DEV** [comment]
> Line 61 and 70, added code to generate random family name.

Please outline the Problem, what you did to solve it, and if the test worked/any straggling bugs.

File: modules/names.py [hunk 1]
```diff
@@ -58,6 +58,7 @@ def get_elf_name(gender):
             for name in elf_male_names:
                 print(name)
             name = input("> ")
+            family_name = random.choice(elf_family_names)
```
Comment (W5DEV) [inline modules/names.py:61]: This should be between line 64 and 65

File: modules/names.py [hunk 2]
```diff
@@ -66,6 +67,7 @@ def get_elf_name(gender):
             for name in elf_female_names:
                 print(name)
             name = input("> ")
+            family_name = random.choice(elf_family_names)
```
Comment (W5DEV) [inline modules/names.py:70]: This should be below the while loop

Comment (Jedikay) [reply modules/names.py:70]: Corrected

**Review by W5DEV — APPROVED** [review]

**Pull request merged**